### PR TITLE
Access Control Fix

### PR DESCRIPTION
**Pull request by KyleD0711**
Added User Role and Role routes, controllers, and models to backend.
Added isAdmin authorization function to secure backend.

### CODE REVIEW

File: app/sequelizeUtils/userrole.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      roleId: userRoleData.roleId,
+    };
+
+    // Save userrole in the database
+    return await UserRole.create(userrole);
+  }
+};
+
+exports.findOneUserRole = async (id) => {
+  return await PersonRole.findByPk(id);
```
Comment (JonathanBooker912) [inline app/sequelizeUtils/userrole.js]: 🤔

**Review by JonathanBooker912 — CHANGES_REQUESTED** [review]
Some Issues

File: app/sequelizeUtils/userrole.js (hunk 1 above)
Comment (KyleD0711) [reply app/sequelizeUtils/userrole.js]: Should be fixed

**Review by JonathanBooker912 — APPROVED** [review]
Looks Good

**Pull request merged**